Newsletters: add 205 (2022-06-22)

harding opened the pull request:
- [x] Clients/services section @bitschmidty 
- [x] Releases/RCs and topics @harding

johnzweng commented on _posts/en/newsletters/2022-06-22-newsletter.md:
Small typo in file suffix: *.html**a**
```suggestion
[poelstra timestamping]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-June/020569.html
```

Xekyo commented on _posts/en/newsletters/2022-06-22-newsletter.md:
```diff
@@ -0,0 +1,230 @@
+would make it easier to enable transaction replacement even for
+transactions that don't opt-in to BIP125, links to information about the
+Hertzbleed sidechannel vulnerability, summarizes the conclusion of a
+discussion about time stamping system design, and examines an new
```
```suggestion
discussion about time stamping system design, and examines a new
```

Xekyo commented on _posts/en/newsletters/2022-06-22-newsletter.md:
Would I understand correctly that address reuse exacerbates this? Perhaps we could call that out specifically to add to the well-known reasons for why address reuse is a bad practice.

Xekyo commented on _posts/en/newsletters/2022-06-22-newsletter.md:
Too many witches spoil the potion—that last sentence is a bit hard to parse. Perhaps:

```suggestion
  belongs to the user and the rest belong to other users.  The user then
  creates a signature which provably came from an owner
  of one of the listed UTXOs, but does not reveal which owner created it.
  ```

Xekyo commented on _posts/en/newsletters/2022-06-22-newsletter.md:
```suggestion
  potentially reduce the user's privacy.  He writes, "there is no
```

Xekyo reviewed (commented):
Thanks that was a great read

harding commented:
Made edits (thanks @johnzweng and @Xekyo ), added releases, added topic links (just one this week), and reviewed @bitschmidty 's contribution (thanks!).  Thanks everyone!

bitschmidty commented:
Merged! Thanks @harding @Xekyo @johnzweng for contributing!
